web/src/app/app.component.scss

```diff
@@ -1,32 +1,66 @@
 @import "~@angular/material/theming";
+@import "../assets/scss/variable";
+@import "../styles.scss";
```

Comment:
We are importing styles.scss in `angular.json` so we don't need to import it in every component